<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>字号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFontSize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>字体:</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>175</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>31</y>
     <width>391</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButtonBold">
    <property name="text">
     <string>粗体</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonItalic_2">
    <property name="text">
     <string>斜体</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUnderline_2">
    <property name="text">
     <string>下划线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonColor">
    <property name="text">
     <string>前景色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBGColor">
    <property name="text">
     <string>后景色</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
